README.md

```diff
@@ -87,9 +99,16 @@ public class MessageListener extends ListenerAdapter
 ## More Examples
 We provide a small set of Examples in the [Example Directory](https://github.com/DV8FromTheWorld/JDA/tree/master/src/examples/java).
 
+In addition you can look at the many Discord Bots that were implemented using JDA:
+- [Yui](https://github.com/DV8FromTheWorld/Yui)
+- [Spectra](https://github.com/jagrosh/Spectra)
```

Comment:
probably should change to `[Vortex](https://github.com/jagrosh/Vortex)` because Spectra's repo is still the JDA2 code